bin/git-forgit

```diff
@@ -136,14 +151,14 @@ _forgit_diff() {
     enter_cmd="cd '$repo' && $get_files | xargs -0 $git_diff -U$_forgit_fullscreen_context -- | $_forgit_diff_pager"
     opts="
         $FORGIT_FZF_DEFAULT_OPTS
-        +m -0 --bind=\"enter:execute($enter_cmd | $_forgit_enter_pager)\"
```

Comment:
Is there a reason why you remove `-0` here but not in all other functions?

Reply:
I simply missed them 😅

Reply:
This behavior is now user-pickable across the board: https://github.com/wfxr/forgit/pull/304/files#diff-e7404556b00c1c7af95effe409a0f05ded287236878c22735194ee0bc0b98d11R40

Reply:
Great!